### ISSUE TITLE
Wrong % rep amounts

### ISSUE THREAD

**Issue by AugustoL** [DXgovernance/DAVI#478]
**Description**
Wrong % of REP in actions

On a proposal in dxgov guild in gnosis: https://qa.projectdavi.eth.limo/#/gnosis/0x3f842726188FcD932d43bcA291be28138228e6D9/proposal/0x00514250dbe1f619f4adcc6b90f47be899d5a36bee900d2754c2b4e1dd5ee7f5

![image](https://user-images.githubusercontent.com/7763867/210254014-a8310dd3-9c9b-4eed-9119-bc96c036e9d5.png)

**Steps to Reproduce**
(https://qa.projectdavi.eth.limo/#/gnosis/0x3f842726188FcD932d43bcA291be28138228e6D9/proposal/0x00514250dbe1f619f4adcc6b90f47be899d5a36bee900d2754c2b4e1dd5ee7f5)

**Device Information**

- OS: Ubuntu
- Browser brave
- Version 1.46

**Comment by rossneilson** [comment]
Yep, unfortunately, we deployed the first guilds with rep not transformed to wei. And the maths is interpreting it as infinite instead of a very very large amount more (10^18). Going to close this since displaying it as a factor isn't much better than infinite

**Comment by AugustoL** [comment]
First we should show the absolute amount of REP, and then if the % is not infinity, we should show it as well. This way at least I can see "2000 REP minted to x account"
